src/ServicePulse.Host/vue/src/components/ConfirmDialog.vue

```diff
@@ -0,0 +1,66 @@
+<script setup>
```

Comment:
I looked through vue and found a few controls that we could use.
https://vuejsexamples.com/tag/dialog/